packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -362,38 +407,40 @@ export class BackgroundScript {
       this.favIcons
         .getFavicon(host)
         .then(favicon => {
-          this.tabStates.set(tab, { favicon })
+          this.tabStates.set(tabId, { favicon })
```

Comment:
This favicon feature doesn't seem actually used ... 
If it were ... we should probably make it work only for the top frame